### ISSUE TITLE
Bad class references

### ISSUE THREAD

**Issue by me90** [hazelcast/hazelcast#16811]
**Describe the bug**
Into the file hazelcast-all-4.0.jar the class `com.hazelcast.web.WebFilterConfig` import the non existing class `com.hazelcast.util.StringUtil` 

**Expected behavior**
I want to use the `hazelcast-all-4.0.jar` to do a Filter Based Web Session Replication.

**To Reproduce**

1. Into web application put the `hazelcast-all-4.0` jars in your `WEB-INF/lib` folder.
2. Add the `com.hazelcast.web.WebFilter` file to the `web.xml` file as described in this link https://github.com/hazelcast/hazelcast-wm

**Additional context**

```
GRAVE [Catalina-utility-1] org.apache.catalina.core.StandardContext.filterStart Exception starting filter [hazelcast-filter]
	java.lang.NoClassDefFoundError: com/hazelcast/util/StringUtil
		at com.hazelcast.web.WebFilterConfig.getBoolean(WebFilterConfig.java:201)
		at com.hazelcast.web.WebFilterConfig.create(WebFilterConfig.java:85)
		at com.hazelcast.web.WebFilter.init(WebFilter.java:156)
		at org.apache.catalina.core.ApplicationFilterConfig.initFilter(ApplicationFilterConfig.java:270)
		at org.apache.catalina.core.ApplicationFilterConfig.getFilter(ApplicationFilterConfig.java:251)
		at org.apache.catalina.core.ApplicationFilterConfig.<init>(ApplicationFilterConfig.java:102)
		at org.apache.catalina.core.StandardContext.filterStart(StandardContext.java:4533)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5172)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:717)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:690)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
		at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:978)
		at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1849)
		at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
		at java.util.concurrent.FutureTask.run(FutureTask.java:266)
		at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
		at java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:112)
		at org.apache.catalina.startup.HostConfig.deployWARs(HostConfig.java:773)
		at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:427)
		at org.apache.catalina.startup.HostConfig.check(HostConfig.java:1620)
		at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:305)
		at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:123)
		at org.apache.catalina.core.ContainerBase.backgroundProcess(ContainerBase.java:1151)
		at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.processChildren(ContainerBase.java:1353)
		at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.processChildren(ContainerBase.java:1357)
		at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.run(ContainerBase.java:1335)
		at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
		at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
		at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
		at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
		at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
		at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.lang.Thread.run(Thread.java:748)
	Caused by: java.lang.ClassNotFoundException: com.hazelcast.util.StringUtil
		at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1365)
		at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1188)
		... 35 more
```

**Comment by enozcan** [comment]
Hi @me90,
`Hazelcast-WM v4.0` (4.0 compatible version) is not yet included in hazelcast-all-4.0.jar. You can use `hazelcast-wm-4.0.jar` (see [releases](https://github.com/hazelcast/hazelcast-wm/releases)) along with `hazelcast-4.0.jar` for now. If you want to use `hazelcast-all`, you need to wait for 4.0.1 release.

**Comment by mesutcelik** [comment]
follow-up issue: https://github.com/hazelcast/hazelcast-wm/issues/85
